purchase_security/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * João Marques
```

Comment:
You can add yourself as contributor if you want to (you can remark that you do the migration from v13 -> v14).